inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp

```diff
@@ -0,0 +1,610 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "mkldnn_strided_slice_node.h"
+
+#include <mkldnn_types.h>
+#include <mkldnn_extension_utils.h>
+
+#include <legacy/ie_layers.h>
+#include "ie_parallel.hpp"
+#include "caseless.hpp"
+#include "common/dnnl_thread.hpp"
+#include "common/cpu_memcpy.h"
+#include "common/dnnl_thread.hpp"
```

Comment:
why do we need dnnl_thread?

Reply:
We need `dnnl_get_max_threads()` for [search](https://github.com/a-sidorova/openvino/blob/51559defd5c6afa0f17c1a5de64ee3df7b7ef531/inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp#L474) optimal dimensions for the best performance.  And we must be sure in [thread count](https://github.com/a-sidorova/openvino/blob/51559defd5c6afa0f17c1a5de64ee3df7b7ef531/inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp#L489) during exectuion because  we calculate indices by thread count before execution